plugins/generic/externalFeed/ExternalFeedBlockPlugin.inc.php

```diff
@@ -36,50 +36,57 @@ function getHideManagement() {
 	 * its category.
 	 * @return String name of plugin
 	 */
-	function getName() {
+	public function getName() {
 		return 'ExternalFeedBlockPlugin';
 	}
 
 	/**
 	 * Get the display name of this plugin.
 	 * @return String
 	 */
-	function getDisplayName() {
+	public function getDisplayName() {
 		return __('plugins.generic.externalFeed.block.displayName');
 	}
 
 	/**
 	 * Get a description of the plugin.
 	 */
-	function getDescription() {
+	public function getDescription() {
 		return __('plugins.generic.externalFeed.description');
 	}
 
 	/**
 	 * Get the external feed plugin
 	 * @return object
 	 */
-	function &getExternalFeedPlugin() {
-		$plugin =& PluginRegistry::getPlugin('generic', $this->parentPluginName);
-		return $plugin;
+	public function getExternalFeedPlugin() {
+		return $this->parentPlugin;
 	}
 
 	/**
 	 * Override the builtin to get the correct plugin path.
 	 * @return string
 	 */
-	function getPluginPath() {
+	public function getPluginPath() {
 		$plugin =& $this->getExternalFeedPlugin();
```

Comment:
`getExternalFeedPlugin` no longer returns a reference, so you'll need to remove the reference here too.